### DISCUSSION TITLE
from php 7.3 to 7.4: array_key_exists(): Using array_key_exists() on objects is deprecated. Use isset() or property_exists() instea

### DISCUSSION THREAD

**Discussion by supersophie** [xenocrat/chyrp-lite#149] [Q&A] [answered]
Hi guys,

I upgraded from Debian 10 to 11  last week and php 7.3 was removed.  My web sites now run 7.4. 

Unforunatly this broke my Chyrp running sqlite.  The page threw a:

array_key_exists(): Using array_key_exists() on objects is deprecated. Use isset() or property_exists() instead.

There is an UPGRADE.md file in the webroot that says the version Sind/2018.0.

How should I get php 7.4 working with Chyrp?

Thanks, S

**Comment by xenocrat** [comment, marked as answer]
You should browse the [release list](https://github.com/xenocrat/chyrp-lite/releases), read the notes and look for a more recent version that claims to fix issues with PHP 7.4, like [this one](https://github.com/xenocrat/chyrp-lite/releases/tag/v2020.02).

**Reply by supersophie** [reply]
Thanks.  I'll install this version and see what happens.

**Reply by supersophie** [reply]
Upgraded and back in action.  Thank-you so much.
